package.json

```diff
@@ -173,7 +173,7 @@
 		"klaw-sync": "^6.0.0",
 		"mdast": "^3.0.0",
 		"moize": "^6.1.0",
-		"next": "^12.3.1",
+		"next": "13.0.2",
```

Comment:
Pinning to 13.0.2 for now as there is a bug with the minifier in 13.0.3 that increased bundle size by a lot: https://github.com/vercel/next.js/issues/42847

Reply:
[v13.0.5 is out!](https://github.com/vercel/next.js/releases/tag/v13.0.5)

Reply:
@dstaley yeah! The most recent commit here is on the latest 👍